bin/internal/README.md

```diff
@@ -0,0 +1,6 @@
+Dart SDK dependency
+===
+
+Dart SDK is downloaded from one of [the supported channels](https://www.dartlang.org/install/archive), cached in `bin/cache/dart-sdk` and is used to run Flutter Dart code.
+
+File `bin/internal/dart-sdk.version` determines the version of Dart SDK that will be downloaded. Normally it points to the `dev` channel(for example, `1.24.0-dev.6.7`), but it can also point to particular bleeding edge build of Dart(for example, `hash/c0617d20158955d99d6447036237fe2639ba088c`).
```

Comment:
seems good.

Some nits:
- have the `===` underline match the heading length, so it looks pretty
- the two paragraphs should start with the word "The" for grammar correctness.
- you missed a space before the two open parentheses in the last paragraph.
- maybe wrap at 80 chars in case someone is loading it in an editor that doesn't wrap.